pythonforandroid/recipes/python3/__init__.py

```diff
@@ -143,9 +143,15 @@ class Python3Recipe(TargetPythonRecipe):
         '__pycache__',
         'tests'
     }
-    '''The directories from site packages dir that we don't want to be included
+    '''the directories from site packages dir that we don't want to be included
     in our python bundle.'''
 
+    site_packages_excluded_dir_exceptions = [
+        'numpy',
+    ]
```

Comment:
I see a discussion about that, can we add a comment that explains why is that needed?